ignite/services/network/node.go

```diff
@@ -77,3 +59,23 @@ func (n Node) StakingParams(ctx context.Context) (stakingtypes.Params, error) {
 	}
 	return res.Params, nil
 }
+
+// IBCInfo Fetches the consensus state with the validator set and the unbounding time
+func (n Node) IBCInfo(ctx context.Context) (networktypes.IBCInfo, int64, error) {
+	status, err := n.cosmos.Status(ctx)
+	if err != nil {
+		return networktypes.IBCInfo{}, 0, err
+	}
+	lastBlockHeight := status.SyncInfo.LatestBlockHeight
+
+	info, err := NodeInfo(ctx, n.cosmos, lastBlockHeight)
+	if err != nil {
+		return networktypes.IBCInfo{}, 0, err
+	}
+
+	stakingParams, err := n.StakingParams(ctx)
+	if err != nil {
+		return networktypes.IBCInfo{}, 0, err
+	}
+	return info, int64(stakingParams.UnbondingTime.Seconds()), nil
```

Comment:
```suggestion
	return info, uint64(stakingParams.UnbondingTime.Seconds()), nil
```

Reply:
Looks like some code duplicated between here and Netowkr.IBCInfo func, can we refactor?

Reply:
it's a little bit different, the new chain gets the height from the last block height from the status method, the SPN gets the height from the chain reward store